src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -275,6 +319,11 @@ public boolean isMasked()
         return mask != null;
     }
 
+    public boolean hasConstraint()
+    {
+        return !columnConstraints.isEmpty();
```

Comment:
@bbotella can not this be "cached"? A simple private boolean in ColumnMetadata saying if constraints are empty or not so we dont need to call this method every time ...